pages/linux/aplay.md

```diff
@@ -0,0 +1,23 @@
+# aplay
+
+> Command-line sound recorder (arecord) and player (aplay) for ALSA soundcard driver.
+
+- Play file "foobar". Sampling rate, bit depth, and so forth will be automatically determined for supported soundfile formats:
+
+`aplay {{foobar}}`
+
+- Play the raw file "foobar" as a 22050-Hz, mono, 8-bit, Mu-Law .au file:
+
+`aplay -c 1 -t raw -r 22050 -f mu_law {{foobar}}`
+
+- Record foobar.wav as a 10-second, CD-quality wave file, using the PCM "copy":
+
+`arecord -d 10 -f cd -t wav -D copy {{foobar.wav}}`
```

Comment:
Hi @NikonP you should add another page for `arecord` because the user interface is `tldr {{command}}`, so the user should use `tldr aplay` to see the `aplay.md` and `tldr arecord` to see examples for the `arecord` command.